appveyor.yml

```diff
@@ -2,18 +2,48 @@ version: 1.{build}
 
 environment:
   matrix:
-    - VisualStudioVersion: 14.0
-      platform: "Win32"
+    - VisualStudio: 14.0
+      VisualStudioGenerator: "14 2015"
+      ConfigurationPlatform: "Win32"
```

Comment:
I had to rename some variables because they conflicted with the ones set by `vcvarsall.bat`.